ragna/ui/api_wrapper.py

```diff
@@ -0,0 +1,108 @@
+import os
+import re
+from datetime import datetime
+
+import emoji
+
+import httpx
+
+
+# The goal is this class is to provide ready-to-use functions to interact with the API
+class ApiWrapper:
+    def __init__(self, api_url):
+        # FIXME: this should be an async client
+        self.client = httpx.Client(base_url=api_url)
```

Comment:
Why do we have any sync functions on the UI side anyway. Shouldn't everything by async?